lib/danger/runner.rb

```diff
@@ -0,0 +1,34 @@
+module Danger
+  class Runner < CLAide::Command
+    self.description = 'Run the Dangerfile.'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.